packages/graphql-transformers-e2e-tests/src/__tests__/SubscriptionsRuntimeFiltering.e2e.test.ts

```diff
@@ -505,17 +514,36 @@ test('Multiple owners auth is supported for subscriptions', async () => {
   }) as unknown as Observable<any>;
   let subscription: ZenObservable.Subscription;
   const subscriptionPromise = new Promise((resolve, _) => {
-    subscription = failedObserver.subscribe(
-      event => {},
-      err => {
-        expect(err.error.errors[0].message).toEqual(
-          'Connection failed: {"errors":[{"errorType":"Unauthorized","message":"Not Authorized to access onCreateTaskGroup on type TaskGroup"}]}',
-        );
+    subscription = observer.subscribe(
```

Comment:
Can we add couple more e2es around:
1. If I am not part of the Dynamic group, I shouldn't be allowed to subscribe.
2. If I have a filter input in addition to dynamic group auth, we appropriately filter the subscriptions.